packages/uniforms-bootstrap4/src/components/fields/ListAddField.js

```diff
@@ -24,4 +24,8 @@ const ListAdd = ({
     );
 };
 
+ListAdd.defaultProps = {
+    addIcon: (<i className="octicon octicon-plus" />),
```

Comment:
sure thing, we each have our own style, I'll try to match yours. FYI my thought process behind the comma: I prefer commas after the final line so when you add a property at the end of an object, the commit is only 1 line change, not 2.